Resources/config/services.yaml

```diff
@@ -13,16 +13,16 @@ services:
             - '../../src/DependencyInjection/'
             - '../../src/Entity/'
             - '../../src/Resources/'
-            - '../../src/OpenIndex.php'
+            - '../../src/OpenIndexExtension.php'
```

Comment:
Interesting, how did this work in the first place?

Reply:
wss door /DependencyInjection waar die file in zit, ook nog OpenIndexExtension.php toevoegen hier is eigenlijk niet nodig denk ik 🤔